TramsDataApi/DatabaseModels/A2BApplicationApplyingSchool.cs

```diff
@@ -88,8 +89,8 @@ public class A2BApplicationApplyingSchool
         public bool? SchoolFinancialInvestigations {get; set;} // int?
         public string SchoolFinancialInvestigationsExplain {get; set;}
         public bool? SchoolFinancialInvestigationsTrustAware {get; set;}
-        public bool? SchoolLoanExists {get; set;}
-        public bool? SchoolLeaseExists {get; set;}        
+        public ICollection<A2BSchoolLease> SchoolLeases { get; set; }
```

Comment:
Just a db model convention, the collections of nested entities should be at the bottom of the model

Reply:
done